dbt/models/default/default.vw_pin_sale.sql

```diff
@@ -249,93 +244,166 @@ sales_val AS (
         sf.run_id AS sv_run_id,
         sf.version AS sv_version
     FROM
-        {{ source('sale', 'flag') }}
-            AS sf
+        {{ source('sale', 'flag') }} AS sf
     INNER JOIN max_version_flag AS mv
         ON sf.meta_sale_document_num = mv.meta_sale_document_num
         AND sf.version = mv.max_version
+),
+
+combined_sales AS (
+    SELECT
+        COALESCE(uq_sales.pin, md_sales.pin) AS pin,
+        COALESCE(uq_sales.year, md_sales.year) AS year,
+        COALESCE(uq_sales.township_code, tc.township_code) AS township_code,
+        COALESCE(uq_sales.nbhd, tc.nbhd) AS nbhd,
+        COALESCE(uq_sales.class, tc.class) AS class,
+        COALESCE(uq_sales.sale_date, md_sales.sale_date) AS sale_date,
```

Comment:
I think here we're going to want to favor `md_sales.date` before 2021, and `uq_sales.sale_date` during and after 2021.